ci-operator/step-registry/windows/e2e/operator/test/windows-e2e-operator-test-ref.yaml

```diff
@@ -10,6 +10,8 @@ ref:
     - name: WINDOWS_SERVER_VERSION
       default: ""
       documentation: This is an optional parameter for the Windows Server version identifier. See https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/base-image-lifecycle. Default is empty and WMCO e2e test suite is responsible for selecting the Windows Server version.
+    - name: COMMUNITY
+      default: "false"
```

Comment:
Consider adding `documentation` field for this var

Reply:
Added